CMakeLists.txt

```diff
@@ -20,18 +20,30 @@ cmake_minimum_required(VERSION 3.6)
 project(hypervisor)
 
 # ------------------------------------------------------------------------------
-# Cmake global properties
+# Config
 # ------------------------------------------------------------------------------
 
 include(scripts/cmake/config/default.cmake)
+include_external_config()
```

Comment:
I think this needs to be called right *before* default.cmake is included to play nicely with the add_config macro, and all of the default values.